examples/destination_adobe_analytics.tf

```diff
@@ -0,0 +1,70 @@
+resource "rudderstack_destination_adobe_analytics" "example" {
+    name = "my-adobe-analytics"
+
+    config {
+        tracking_server_url = "http://sampleurl.com"
+        report_suite_id = "id01, id02"
+        # heartbeat_tracking_server_url = "http://sampleheartbeaturl.com"
+        # events_to_types = [{
+		# 	from = "video start"
+		# 	to = "heartbeatPlaybackStarted"
+		# }]
+		# list_delimiter = [{
+		# 	from = "listPhone"
+		# 	to = ","
+		# }]
+		# props_delimiter = [{
+		# 	from = "customPhone"
+		# 	to = ","
+		# }]
+		# event_merch_properties = [{
```

Comment:
Can we simplify `event_merch_properties` and `product_merch_properties` to string arrays?
An example of how it can be done with `ArrayWithStrings` is here: https://github.com/rudderlabs/terraform-provider-rudderstack/blob/5cacb4ca77010ddb826658b5be33226148d2d6b3/rudderstack/integrations/destinations/destination_amplitude.go#L22

Reply:
Addreseed